Description: fix return type of gtk_completion_line_get_type
 Patch from fedora was slightly modified (parts dropped, fuzz fixed) to apply on
 top of Debian package by Andreas Henriksson <andreas@fatal.se>
Origin: https://src.fedoraproject.org/cgit/rpms/gmrun.git/plain/gmrun-0.9.2-f12.patch
Bug-Debian: https://bugs.debian.org/857065

--- a/src/gtkcompletionline.cc
+++ b/src/gtkcompletionline.cc
@@ -77,9 +77,9 @@
 on_key_press(GtkCompletionLine *cl, GdkEventKey *event, gpointer data);
 
 /* get_type */
-guint gtk_completion_line_get_type(void)
+GtkType gtk_completion_line_get_type(void)
 {
-  static guint type = 0;
+  static GtkType type = 0;
   if (type == 0)
   {
     GtkTypeInfo type_info =
--- a/src/gtkcompletionline.h
+++ b/src/gtkcompletionline.h
@@ -76,7 +76,7 @@
     void (* cancel)(GtkCompletionLine *cl);
   };
 
-  guint gtk_completion_line_get_type(void);
+  GtkType gtk_completion_line_get_type(void);
   GtkWidget *gtk_completion_line_new();
 
   void gtk_completion_line_last_history_item(GtkCompletionLine*);
